Dockerfile

```diff
@@ -1,27 +1,47 @@
-FROM python:3.8-slim
-
-RUN apt-get update && apt-get -y install git gcc
-
-RUN useradd -r app_user
-RUN mkdir /app
-RUN chown app_user /app/
-WORKDIR /app
+FROM python:3.10-alpine
+
+ENV PYTHONPATH="/usr/local/src/idunn/src"
+ENV PYTHONUNBUFFERED=1
+
+# create the user idunn
+RUN mkdir /usr/local/src \
+ && addgroup --gid 1000 idunn \
+ && adduser \
+      --disabled-password \
+      --home /usr/local/src/idunn \
+      --ingroup idunn \
+      --shell /bin/sh \
+      --uid 1000 \
+      idunn
+
+RUN apk update \
+    && apk add --upgrade --no-cache \
+        bash openssh curl ca-certificates openssl less htop gcc git \
+		g++ make wget rsync \
+        build-base libpng-dev freetype-dev libexecinfo-dev openblas-dev libgomp lapack-dev \
+		libgcc libquadmath musl  \
+		libgfortran \
+		lapack-dev \
```

Comment:
How did you build this list of required packages? Some of them like `less` or `htop` seem dubious here :thinking:

Reply:
this part I have just copy pasted let's see if we can remove something. The most annyoing dependency was spacy